lib/Cake/Test/Case/Utility/FileTest.php

```diff
@@ -62,6 +63,15 @@ public function tearDown() {
  * testBasic method
  *
  * @return void
+ * @covers ::__construct
+ * @covers ::info
+ * @covers ::ext
+ * @covers ::name
+ * @covers ::md5
+ * @covers ::size
+ * @covers ::owner
+ * @covers ::group
+ * @covers ::Folder
```

Comment:
The PHPUnit docs say this form is for [global functions](https://phpunit.de/manual/3.7/en/appendixes.annotations.html#appendixes.annotations.covers) unless I'm misreading the docs.